### PR TITLE
[13.0][MIG] stock_request_tier_validation

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard migration.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, thanks!

File: stock_request_tier_validation/README.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Configuration
+=============
+
+A default tier is created allowing Stock Request Manager to approve Stock
+Request and Stock Request Orders.
+
+In addition, you may want to add more tiers, so:
+
+#. Go to *Settings > Technical > Tier Validations > Tier Definition*.
+#. Create as many tiers as you want for Purchase Order model.
```
Comment (lfreeke) [inline stock_request_tier_validation/README.rst]: I think this should be 'Stock Request' model.

Comment (ps-tubtim) [reply stock_request_tier_validation/README.rst]: Thank you for review. I edited this file.

**Comment by etobella** [comment]
Can you cherry-pick #869

**Comment by LoisRForgeFlow** [comment]
@lfreeke @etobella is this ok for you?

File: stock_request_tier_validation/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019-2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Request Tier Validation",
+    "summary": "Extends the functionality of Stock Requests to "
+    "support a tier validation process.",
+    "version": "13.0.1.0.0",
+    "category": "Warehouse Management",
```
Comment (rousseldenis) [inline stock_request_tier_validation/__manifest__.py]: Change this to 'Warehouse' as it won't appear in good category on OCA shop

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comment

**Comment by LoisRForgeFlow** [comment]
@ps-tubtim seems there is a merge commit in the history, can you try to clean that? Thanks

File: stock_request_tier_validation/__manifest__.py (hunk 2 above)
Comment (ps-tubtim) [reply stock_request_tier_validation/__manifest__.py]: Changed.

**Comment by LoisRForgeFlow** [comment]
pinging again @lfreeke and @etobella:  is this ok for you?

**Comment by lfreeke** [comment]
@LoisRForgeFlow :+1:

**Pull request merged**